pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -61,7 +66,13 @@ func SetupNetworkInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain)
 			return err
 		}
 
-		err = vif.Plug(&iface, network, domain)
+		if networks[iface.Name].Multus != nil {
+			podInterfaceName = fmt.Sprintf("net%d", multusNetworks[iface.Name])
```

Comment:
can't the interface name of the multus network be named anything?  How are we guaranteed that the interface is named using the convention here?